docs/resources/architecture/themes.mdx

```diff
@@ -0,0 +1,193 @@
+
+The `css` function can be used to create reusable styles through the application of a class name or used inline to style individual elements. Here’s an example of inline use, utilizing the already defined `FancyButton` from before:
+
+```jsx
+<FancyButton css={{ //base styles, optional variants }}>I'm fancy!</FancyButton>
+```
+
+And below demonstrates the application of css through the classname property:
+
+```jsx
+const newButton = css({
+  // base styles
+});
+
+// Use it
+<div className={newButton()}>I'm new!</div>;
+```
+
+\*\*note: style property names use camel case in these functions (example: `z-index` becomes `zIndex`)
+
+## Keyframes
+
+Another function available in Stitches is the `keyframes` function. Using the keyframe animation technique, Stitches allows the creation of global CSS keyframe rules, which let transitions be easily reused across code. Below is an example of a simple fading-out animation:
+
+```jsx
+import { keyframes } from '@washingtonpost/wpds-ui-kit'
+
+const fadeOut = keyframes({
+	from: { opacity: 1 },
+	to: { opacity: 0 },
+});
+
+// Use it
+const fadingInButton = styled("button", {
+	'&hover': {
+		animation: `${fadeOut} 200ms ease-out`,
+	}
+)};
+```
+
+---
+
+## Theme object and tokens
+
+The theme object contains useful token data that can be applied when styling an element. WPDS specifies its own tokens for many attributes, which can be explored throughout the [foundations section](https://build.washingtonpost.com/#Foundations) of our documentation. Tokens can be accessed in code by using the `theme` object, which is is preferred over using the `$` reference symbol because it provides additional checks. For more information about using our theme tokens, check out [this tutorial](https://build.washingtonpost.com/resources/tutorials/how-to-use-our-theme).\
+The `const WPDS` is pre-defined [here](https://github.com/washingtonpost/wpds-ui-kit/blob/main/ui/theme/src/stitches.config.ts).
+
+```jsx
+const WPDS = stitches.createStitches({
+  prefix,
+  theme: {
+    colors: {
+      ...tokens.light,
+      ...tokens.staticColors,
+      ...tokens.defaultTheme,
+    },
+    sizes: tokens.sizes,
+    ...
+  },
+  media: { ... },
+  utils: { ... },
+});
+
+// Use it
+const BestButton = styled('button', {
+	// use the size token $200, which is equivalent to 2rem or 32px
+	marginTop: theme.space["200"],
+	// use the color token $primary, which is equivalent to rgba(17, 17, 17, 1)
+	// color tokens change automatically when a color theme is applied
+	color: theme.colors.primary,
+});
+```
+
+## Breakpoints
+
+Stitches is also capable of defining breakpoints, which are reusable media queries useful for responsive styling. The WPDS UI Kit supports a variety of size breakpoints and custom themes, which can be explored [here](https://github.com/washingtonpost/wpds-ui-kit/blob/main/ui/theme/src/stitches.config.ts). The button in the example below would be uniquely styled when the viewport is smaller than `768px`, or `sm`.
```

Comment:
Personally, I'm not a big fan of having the word `here` be a link out. Perhaps you could say 'which can be explored in the [stitches.config.ts](https://github.com/washingtonpost/wpds-ui-kit/blob/main/ui/theme/src/stitches.config.ts)' or something like that